app/controllers/user_applications_controller.rb

```diff
@@ -171,13 +180,41 @@ def get_application_volunteer_choices
     @third_choice_volunteer_position = VolunteerPosition.find(@user_application.third_choice_volunteer_position_id) unless @user_application.third_choice_volunteer_position_id.nil?
   end
 
-  def change_status(params, status, message)
-    @user_application = UserApplication.find(params['id'])
-    authorize @user_application
+  def change_status(status, message)
     @user_application.user_application_status_id = UserApplicationStatus.find_by(status: status).id
     @user_application.status_changed_by = @user.id
+
     if @user_application.save
       redirect_to user_applications_path, :alert => message
     end
   end
+
+  def set_user_applications_joins_users
+    return "inner join users on users.id = user_applications.user_id"
```

Comment:
@SeanLF same - you can get rid of all these "return" keywords and just put the string.